graphql_compiler/compiler/ir_lowering_sql/constants.py

```diff
@@ -20,13 +20,47 @@
 
 SUPPORTED_BLOCK_TYPES = (
     blocks.QueryRoot,
+    blocks.Filter,
 )
 
 SUPPORTED_OUTPUT_EXPRESSION_TYPES = (
     expressions.OutputContextField,
 )
 
 
+class Operator(object):
```

Comment:
How do you feel about making this a `namedtuple`? That way, you get defined fields, immutability, and auto-defined operators like `==`.

Reply:
Done.